deploy/olm-catalog/nxiq-operator-certified/1.94.0-1/nxiq-operator-certified.v1.94.0-1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,368 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "sonatype.com/v1alpha1",
```

Comment:
presumably we don't want 'alpha' in releases?

Reply:
This has been a convention for all the previous versions of the operator so far. as you can see above, the entire operators api for ClusterServiceVersion is "v1alpha1", so we're in good company, I guess. There has been some discussion of changing some of these alpha and beta references (elsewhere) if needed for the next steps in IBM certification. I'm not sure it's required yet. I've not felt bad about leaving these alpha/beta, because it truthful about potential changes, especially since there are still so many fixes we *could* continue to make to these configs.